spec/features/item_spec.rb

```diff
@@ -74,35 +74,53 @@
     expect(page.all("select#filters_in_category option").map(&:text).select(&:present?)).not_to eq(expected_order.reverse)
   end
 
-  describe "Item Table Tabs >" do
-    let(:item_name_1) { "the most wonderful magical pullups that truly potty train" }
-    let(:item_name_2) { "blackbeard's rugged tampons" }
-    before :each do
-      @item = create(:item, name: item_name_1, category: "same")
-      @item2 = create(:item, name: item_name_2, category: "different")
-      @storage = create(:storage_location, :with_items, item: @item, item_quantity: 666, name: "Test storage")
+  describe "Item Table Tabs >", focus: true do
+    let(:item_pullups) { create(:item, name: "the most wonderful magical pullups that truly potty train", category: "Magic Toddlers") }
+    let(:item_tampons) { create(:item, name: "blackbeard's rugged tampons", category: "Menstrual Products") }
+    let(:storage_name) { "the poop catcher warehouse" }
+    let(:storage) { create(:storage_location, :with_items, item: item_pullups, item_quantity: num_pullups_in_donation, name: storage_name) }
+    let!(:aux_storage) { create(:storage_location, :with_items, item: item_pullups, item_quantity: num_pullups_second_donation, name: "a secret secondary location") }
+    let(:num_pullups_in_donation) { 666 }
+    let(:num_pullups_second_donation) { 1 }
+    let(:num_tampons_in_donation) { 42 }
+    let(:num_tampons_second_donation) { 17 }
+    let(:donation_tampons) { create(:donation, :with_items, storage_location: storage, item_quantity: num_tampons_in_donation, item: item_tampons) }
+    let(:donation_aux_tampons) { create(:donation, :with_items, storage_location: aux_storage, item_quantity: num_tampons_second_donation, item: item_tampons) }
+    before do
+      storage.intake!(donation_tampons)
+      aux_storage.intake!(donation_aux_tampons)
       visit url_prefix + "/items"
     end
     # Consolidated these into one to reduce the setup/teardown
     scenario "Displays items in separate tabs", js: true do
-      expect(page.find("table#tbl_items", visible: true)).not_to have_content "Quantity"
-      expect(page.find(:css, "table#tbl_items", visible: true)).to have_content(@item.name)
-      expect(page.body).to include(item_name_1)
-      expect(page.body).to include(item_name_2)
+      tab_items_only_text = page.find("table#tbl_items", visible: true).text
```

Comment:
I want to avoid expensive Capybara calls, so I'm avoiding calling duplicate `page.find`s.
Please share suggestions for any Capybara performance improvements.

Reply:
Suggestion from Jamie:
```
page.find('the thing').tap do |element|
  expect(element).to have_content 'x'
  expect(element).to have_content 'y'
end
```

Reply:
Upon some research, looks like there is an `.and` rspec operator. So maybe you can do:

    expect(element).to have_content('x').and have_content('y').and have_content('z')

I have not tried this. Via https://relishapp.com/rspec/rspec-expectations/docs/compound-expectations